packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -35,6 +36,13 @@ export class SwatchColorPickerBase extends React.Component<ISwatchColorPickerPro
   private readonly navigationIdleDelay: number = 250 /* ms */;
   private async: Async;
 
+  // Add an index to each color cells. Memoizes this so that color cells do not re-render on every update.
+  private _getItemsWithIndex = memoizeFunction((items: IColorCellProps[]) => {
```

Comment:
This should typically be below the constructor and public methods--did the linter not like that due to the memoize wrapper?

Reply:
Yep. It complained that this is a private field, so it should be before the constructor.

Reply:
Since, I cannot do much here :) resolving this.